eng/testing/AndroidRunnerTemplate.sh

```diff
@@ -16,22 +21,19 @@ while true; do
     fi
 done
 
-if [ -z "$HELIX_WORKITEM_UPLOAD_ROOT" ]; then
-    XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
-else
-    XHARNESS_OUT="$HELIX_WORKITEM_UPLOAD_ROOT/xharness-output"
-fi
-
 if [ ! -z "$XHARNESS_CLI_PATH" ]; then
```

Comment:
Do we still need this though?

Reply:
(and same for iOS)

Reply:
Yes we use this locally to override the xharness CLI path when trying out local xharness builds.